### ISSUE TITLE
Tailscale admin console doesn't recognize TS clients above 1.100

### ISSUE THREAD

**Issue by jzebedee** [tailscale/tailscale#20238]
### What is the issue?

The Tailscale Admin Console's version checking logic doesn't seem to work for version fragments of more than two digits. I noticed it in the TLS Certificate Status reporting, but I'm not sure if it's used elsewhere.

<img width="1082" height="634" src="https://github.com/user-attachments/assets/ce7d7adf-5546-4250-88b8-633fb5b0e23e" />

### Steps to reproduce

1. View Machine Details for a node with TS client version 1.9x (e.g., `1.98.5`) and TLS status checking works normally
2. Upgrade to TS client version 1.10x (e.g., `1.100.0`)
3. TLS status checking breaks with "Upgrade client to check status" / "TLS Certificate Status Reporting was added in Tailscale v1.56"

### Are there any recent changes that introduced the issue?

First seen on `dhi.io/tailscale:1` OCI when it rolled over to TS client version `1.100`

### OS

Linux, Windows, Android

### OS version

6.18.33.1-microsoft-standard-WSL2

### Tailscale version

1.100.0

**Comment by alexwlchan** [comment]
Good catch, thanks for raising! The fix is quite straightforward; I'll pick it up now.

**Comment by alexwlchan** [comment]
This should be fixed now, thanks for flagging!
